src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -154,6 +154,7 @@ public boolean handleInput(String input, boolean needsDetail, boolean confirmed,
             if (screen instanceof EntityScreen) {
                 EntityScreen entityScreen = (EntityScreen)screen;
                 boolean autoLaunch = entityScreen.getAutoLaunchAction() != null;
+                addBreadcrumb = !autoLaunch;
```

Comment:
While debugging something else, I realized breadcrumbs are still off. What's interesting is that when I select a case from [this search-first menu](http://localhost:8000/a/bosco/apps/view/f6d2aa801557aaae7b2c1b180600484a/module/96cb622b8288af9d9fe3e39e9481717dba2d80dd/), the case name *does* get added to the breadcrumb, but it then gets removed later by `executeAndRebuildSession`.

Reply:
Thanks! It was a [silly mistake ](https://github.com/dimagi/formplayer/pull/843/commits/5cc9af228f0ef284710adcdc2c50f3f0d59ea442)on my side.

Reply:
phew, I was worried that one would be tricky 😅